server/routes/board.js

```diff
@@ -34,22 +37,22 @@ router.post('/', (req, res) => {
    }
 })
 
-// router.post('/', (req, res) => {
-//    try{
+router.post('/drag', (req, res) => {
```

Comment:
it's better maybe to patch? because when u use post it's mean that u want to create new item